docs/installation.md

```diff
@@ -1,3 +1,12 @@
-!!! warning "Page under construction"
+Installation is as simple as:
 
-    This page is under construction. Please check back later. Consider contributing to this page by opening a PR! 
\ No newline at end of file
+```bash
+npm install instructor-js
```

Comment:
not true yet, as we cannot get `instructor` on npm, can you keep it a warning

Reply:
might have to suggest installing from github

Reply:
added